metrics/prometheus_test.go

```diff
@@ -61,3 +68,24 @@ deployment.apps/deploymentName configured (server dry run)`
 		t.Error(diff)
 	}
 }
+
+func TestOutputWarningLine(t *testing.T) {
```

Comment:
For the sake of consistency with the other tests, this function could be called `TestParseKubectlOutputWarningLine`. To be honest, `TestParseKubectlServerDryRunOutput` should probably be `TestParseKubectlOutputServerDryRun` as well.